src/test/suite/fixtures/recorded/inference/bringHarpAfterLineTrapAndBlockSpun.yml

```diff
@@ -0,0 +1,62 @@
+languageId: plaintext
```

Comment:
This test is the punchline: having "block" as a modifier on the second destination target still allows it to infer "after" from the first destination target